src/IssueCreator/Settings.cs

```diff
@@ -104,12 +129,31 @@ public Settings Clone()
             Settings newSettings = new Settings()
             {
                 Repositories = new List<string>(this.Repositories),
+                SelectedRepository = string.Copy(this.SelectedRepository),
```

Comment:
Why not just assigning the value?

Reply:
Initially I was planning to data bind this and so I was being careful to isolate the change of the value due to binding, but it's not needed now.